passthrough.go

```diff
@@ -64,18 +65,50 @@ func LeaseSwitchedPassthroughBackend(ctx context.Context, conf *logical.BackendC
 				Operations: map[logical.Operation]framework.OperationHandler{
 					logical.ReadOperation: &framework.PathOperation{
 						Callback: b.handleRead(),
+						Responses: map[int][]framework.Response{
+							http.StatusOK: {{
+								Description: http.StatusText(http.StatusOK),
+								Fields:      nil, // dynamic fields
```

Comment:
how are we handing dynamic fields? (I've been putting [empty struct](https://github.com/hashicorp/vault/pull/18628/files#diff-56d9bad9715ff3f7ee5c46d88094b033f8e4b14f1ef35a285fd0df47af72e610R1563-R1564) and encouraging others to do the same)

Reply:
The validation function currently [checks for the `nil` schema](https://github.com/hashicorp/vault/blob/39211b8772559d626285379b5ad6f619a094137c/sdk/framework/field_data.go#L67-L70). An empty map is somewhat ambiguous, it could mean either:

- this schema must have 0 fields (a more logical interpretation, imo)
- this schema can have any fields

A `nil` map doesn't have that ambiguity and so can be used a "sentinel" value.

Reply:
ok, so use `nil` for dynamically keyed responses now?

Reply:
Yes, please :)